Modules/_testcapimodule.c

```diff
@@ -3272,6 +3272,15 @@ PyInit__testcapi(void)
     PyModule_AddObject(m, "SIZEOF_WCHAR_T", PyLong_FromSsize_t(sizeof(wchar_t)));
     PyModule_AddObject(m, "SIZEOF_VOID_P", PyLong_FromSsize_t(sizeof(void*)));
     PyModule_AddObject(m, "SIZEOF_TIME_T", PyLong_FromSsize_t(sizeof(time_t)));
+    PyModule_AddObject(m, "SIZEOF_INTMAX_T", PyLong_FromSsize_t(sizeof(intmax_t)));
+    PyModule_AddObject(m, "SIZEOF_INTPTR_T", PyLong_FromSsize_t(sizeof(intptr_t)));
+    PyModule_AddObject(m, "SIZEOF_PTRDIFF_T", PyLong_FromSsize_t(sizeof(ptrdiff_t)));
+#ifdef MS_WINDOWS
+    PyModule_AddObject(m, "SIZEOF_OFF_T", PyLong_FromSsize_t(sizeof(long long)));
```

Comment:
It looks like `off_t` is "no narrower than `int`" on all platforms, and on Windows is defined (if you happen to include it) as `long`, which is 32-bits (and no narrower than `int`). Seems there's an optional preprocessor variable on POSIX to get it to be 64-bits, but otherwise it's also `int` and not `long` there.

Perhaps we should define it as the same as `ptrdiff_t` if it's not more available? Or just leave it off on Windows (though I _think_ I'd prefer the portability of having something, even though it's not going to be used for any real purpose).

Reply:
In `Modules/_io/_iomodule.h`, `Modules/posixmodule.c` and `Modules/mmapmodule.c` it is defined as `long long` on Windows and `off_t` on Posix, but there is no globally visible definition.

These tests are remnants from other PR, with specialized code for each type. I leave them just to be sure that the general code supports them.